src/DssSpell.sol

```diff
@@ -41,7 +37,25 @@ contract DssSpellAction is DssAction {
     uint256 constant RAY = 10**27;
     uint256 constant RAD = 10**45;
 
-    address constant MCD_FLASH = 0x1EB4CF3A948E7D72A198fe073cCb8C7a948cD853;
+    bytes32 constant ILK_PSM_USDC_A     = "PSM-USDC-A";
+
+    // Growth Core Unit
+    address constant GRO_MULTISIG        = 0x7800C137A645c07132886539217ce192b9F0528e;
+    // Content Production Core Unit
+    address constant MKT_MULTISIG        = 0xDCAF2C84e1154c8DdD3203880e5db965bfF09B60;
+    // GovAlpha Core Unit
+    address constant GOV_MULTISIG        = 0x01D26f8c5cC009868A4BF66E268c17B057fF7A73;
+    // Real-World Finance Core Unit
+    address constant RWF_MULTISIG        = 0x9e1585d9CA64243CE43D42f7dD7333190F66Ca09;
+    // Risk Core Unit
+    address constant RISK_MULTISIG       = 0xd98ef20520048a35EdA9A202137847A62120d2d9;
+    // Protocol Engineering Multisig
+    address constant PE_MULTISIG         = 0xe2c16c308b843eD02B09156388Cb240cEd58C01c;
+    // Oracles Core Unit (Operating)
+    address constant ORA_MULTISIG        = 0x2d09B7b95f3F312ba6dDfB77bA6971786c5b50Cf;
+    // Oracles Core Unit (Emergency Fund)
+    address constant ORA_ER_MULTISIG     = 0x53CCAA8E3beF14254041500aCC3f1D4edb5B6D24;
```

Comment:
Checked these against values used in a previous spell (https://etherscan.io/address/0x379746b2bd85Bb837D44F5f9299C308F33Ed214A#code), except for the Oracle values, which I confirmed via the forum post and also a direct double-check with Nik. General note for other reviewers: it's possible to view edit history on forum posts, which is important for confirming that no malicious edits have been made.